openerp/tools/config.py

```diff
@@ -281,6 +281,9 @@ def __init__(self, fname=None):
             group.add_option("--limit-request", dest="limit_request", my_default=8192,
                              help="Maximum number of request to be processed per worker (default 8192).",
                              type="int")
+            group.add_option("--limit-socket-timeout", dest="limit_socket_timeout", my_default=-1.0,
+                             help="Maximum time in second to spend in blocking socket operations (default: disabled).",
```

Comment:
- second**s**

- Also it would be useful to mention that this setting only applies with workers>0, and only for HTTP workers